src/screens/users/list/Users.tsx

```diff
@@ -134,46 +145,51 @@ export default class Users extends BaseAutoRefreshScreen<Props, State> {
     await this.refresh();
   };
 
-  public render = () => {
+  public render() {
     const style = computeStyleSheet();
     const { users, count, skip, limit, refreshing, loading } = this.state;
-    const { navigation } = this.props;
+    const { navigation, modal, select, onUserSelected, initiallySelectedUsers } = this.props;
     return (
       <Container style={style.container}>
-        <HeaderComponent
-          title={this.title ?? i18n.t('sidebar.users')}
-          subTitle={count > 0 ? `${I18nManager.formatNumber(count)} ${I18n.t('users.users')}` : null}
-          navigation={this.props.navigation}
-          leftAction={this.onBack}
-          leftActionIcon={'navigate-before'}
-          rightAction={() => {
-            navigation.dispatch(DrawerActions.openDrawer());
-            return true;
-          }}
-          rightActionIcon={'menu'}
-        />
+        {!modal && (
+          <HeaderComponent
+            title={this.title ?? i18n.t('sidebar.users')}
+            subTitle={count > 0 ? `${I18nManager.formatNumber(count)} ${I18n.t('users.users')}` : null}
+            navigation={this.props.navigation}
+            leftAction={this.onBack}
+            leftActionIcon={'navigate-before'}
+            rightAction={() => {
+              navigation.dispatch(DrawerActions.openDrawer());
+              return true;
+            }}
+            rightActionIcon={'menu'}
+          />
+        )}
         {loading ? (
           <Spinner style={style.spinner} color="grey" />
         ) : (
           <View style={style.content}>
             <SimpleSearchComponent onChange={async (searchText) => this.search(searchText)} navigation={navigation} />
             <ItemsList<User>
+              select={select}
+              onSelect={(userIds: User[]) => onUserSelected(userIds)}
```

Comment:
You see, you even named the variable 'userIds' ;-)
And it's userIDs. 
I think you can reference the method right away:
`onSelect={onUserSelected}`